### PR TITLE
Add Result - MAM

### PR DESCRIPTION
**Pull request by jkrvivian**
* channel/endpoint creation
* attach a message through MAM

## TODO: Attaching MAM message on RPi3 is unfinished

### CODE REVIEW

**Comment by jserv** [comment]
Avoid referring to `RPi3`. Instead, always use `Raspberry Pi 3`.

File: paper.tex [hunk 1]
```diff
@@ -417,6 +417,85 @@ \subsubsection{Numerical Example}
 
 
 \section{Results}
+It is important to claim that all participants in data marketplace do not need to hold an IOTA full node which maintains the transaction history and exchanges information of the Tangle. Each role are only required to run client libraries and communicate with IOTA full nodes to interact with the Tangle. Therefore, in the following evaluations, all devices run with client library only.
```
Comment (twzjwang) [inline paper.tex]: Each role "is"

File: paper.tex [hunk 2]
```diff
@@ -417,6 +417,85 @@ \subsubsection{Numerical Example}
...
+% TODO: RPi3 statistics
+
+\begin{figure}[H]
+    \centering
+    \includegraphics[width=4.4 in]{mam_send}
+    \caption{Time cost of sending a message through MAM.}
+    \label{fig:mam_send}
+\end{figure}
+
+MAM operations take a lot of time for both devices, but RPi3 spends even more time than PC. The simulation results above indicate that MAM is difficult for low-level sensor devices to run, which these devices are the majority hardware as data providers in the data marketplace. Furthermore, sensors with the low computing power and weak internet connection are not able to have enough resources to handle data collection, data transmission on MAM and trading process with consumers simultaneously. Therefore, transferring MAM operations to brokers, which can be PCs or even more powerful machines, while ensuring the profit of providers can effectively solve performance problems and lower the threshold for participation in the data marketplace. However, even on PC, MAM operations still cost a considerable time, improving the performance is an essential issue that need to be done for the next step.
```
Comment (twzjwang) [inline paper.tex]: that "needs" to be done

**Comment by jserv** [comment]
Fix the conflicts.

**Pull request merged**